src/logic/wrapper.js

```diff
@@ -7,7 +8,16 @@ module.exports.Wrapper = ({ router, module }) => {
   const wrapFn = (identifier, params, ...args) => {
     assert([1, 2].includes(args.length));
     const [options, handler] = args.length === 2 ? args : [{}, args[0]];
+    // process.env.RATE_LIMIT is a string
```

Comment:
Should we strictly enforce rate limiting being a Number??

Reply:
I think we should enforce as an integer and handle it as needed